diff --git a/dist/cli/build/patches/plugins/load-manifest.js b/dist/cli/build/patches/plugins/load-manifest.js
index bff72bd613f34fc7e2f785f0961c15678d524a71..c6758d696e908fbdbcd824783f8c3a085319d4fa 100644
--- a/dist/cli/build/patches/plugins/load-manifest.js
+++ b/dist/cli/build/patches/plugins/load-manifest.js
@@ -31,7 +31,7 @@ async function getLoadManifestRule(buildOpts) {
     const { outputDir } = buildOpts;
     const baseDir = join(outputDir, "server-functions/default", getPackagePath(buildOpts));
     const dotNextDir = join(baseDir, ".next");
-    const manifests = await glob(join(dotNextDir, "**/{*-manifest,required-server-files,prefetch-hints}.json"), {
+    const manifests = await glob(join(dotNextDir, "**/{*-manifest,required-server-files,prefetch-hints,preview-props}.json"), {
         windowsPathsNoEscape: true,
         nodir: true,
     });
